### ISSUE TITLE
Some kits don't have their content published to ipfs-cache (aragen)

### ISSUE THREAD

**Issue by 0x6431346e** [aragon/aragon-cli#227]
![image](https://user-images.githubusercontent.com/26041347/47156881-b7a79f80-d2f0-11e8-9ba1-5a5580c68c9e.png)
![image](https://user-images.githubusercontent.com/26041347/47156895-c0987100-d2f0-11e8-8b33-d87a327e7a8e.png)

To reproduce:
1. `aragon devchain`
2. `npm run start:local` on the `aragon/aragon` repo
3. Create a new organization

Alternatively:
1. `aragon init foo`
2. `aragon run`
3. Instead of going to your organization, go to the welcome screen and create a new organization

Related to https://github.com/aragon/aragon-cli/issues/177

**Comment by izqui** [comment]
The current version of `aragen` doesn't have the DAO kits deployed, that's why the ENS names cannot be found. `aragen` versions will start containing these very soon.

**Comment by sohkai** [comment]
@0x6431346e With v5.2.0, I think this should be fixed now? I think we can also close #177 since we couldn't reproduce it and it was a fairly long time ago.

**Comment by 0x6431346e** [comment]
I think we can close #177 since we cannot replicate it, but this one is not yet fixed, just tried it with `v5.2.0`

**Comment by sohkai** [comment]
@0x6431346e are those related to #320?

This should be working with the latest aragen.

**Comment by 0x6431346e** [comment]
@sohkai Probably, but it should work at least with `aragon/aragon`, but now it gets:
`Error: Could not fetch ABI for template 'multisig'
    at Object.newDAO (index.js:104)`

In `aragon.js` we have this piece of code:
```
      const { contractAddress, abi } = await apm.getLatestVersion(tmplObj.appId)

      if (!contractAddress) {
        throw new Error(`No contract found on APM for template '${templateName}'`)
      }
      if (!abi) {
        throw new Error(`Could not fetch ABI for template '${templateName}'`)
      }
```
And in the CLI we have this workaround, in `dao new`:
```
          // TODO: Remove hack once https://github.com/aragon/aragen/pull/15 is finished and published
          const abi = ctx.repo.abi || BARE_KIT_ABI
```

Looking into it a bit more, I can see `apm versions` returns:
```
 % aragon apm versions --environment staging                                                 !11009
 ℹ multisig-kit.aragonpm.eth has 1 published versions
 ✔ 1.0.0: 0x01F2e9CDAac527Ffef1B9869d4C0B33357c0A2Bc ipfs:QmXVacmV6q9KRTEhtUfdgHamBN5N2XGMVf254MPerN3HFk
```

```
 % aragon apm versions                                                                       !11012
 ℹ multisig-kit.aragonpm.eth has 1 published versions
 ✖ 1.0.0: 0x5aE47822Da849508020c25E2E92c4a4cC4E03001 Version not found in provider
```

I guess the kits contents are not added to aragon ipfs correctly.

**Comment by sohkai** [comment]
> I guess the kits contents are not added to aragon ipfs correctly.

Very odd... will try to take a look at this and get a handle on aragen. It's likely we need to add those `ipfsCache` files to a local IPFS daemon or something (so they can be resolved).

**Comment by sohkai** [comment]
I just realized this is somewhat painful to solve, and the only two options we really have are:

1. Including the ABIs of the kits published on aragen in the CLI itself (maybe easiest?)
1. Using the same content hash for the kit as on a production environment, where we have servers pinning them (may change in the future and become unresolvable).

I'm leaning on 1, since I don't think we should publish very many kits by default in aragen and hard coding this seems reasonable (we did this for the `BareKit` in #126 originally as a temporary fix).
